docs/src/content/docs/ucs/03.mdx

```diff
@@ -992,6 +991,10 @@ The stake instruction allows cross-chain staking of governance tokens with valid
 3. A staking position NFT is minted to the beneficiary
 4. The NFT represents ownership of the staked position and can be used to manage it
 
+:::note
+The `path` field in the packet must be zero for staking operations. The staked token must match the governance token configured for the channel.
```

Comment:
why do we check against the configured gov token, rather than just use the configured value?